recipe/build-one.bat

```diff
@@ -1,21 +1,32 @@
 setlocal EnableDelayedExpansion
 
+where flang.exe
+if not errorlevel 1 (
+  echo using fortran
+  set "CMAKE_ARGS=!CMAKE_ARGS! -DSUITESPARSE_HAS_FORTRAN:BOOL=ON -DCMAKE_Fortran_COMPILER=flang.exe"
+) else (
+  echo no fortran
+  set "CMAKE_ARGS=!CMAKE_ARGS! -DSUITESPARSE_HAS_FORTRAN:BOOL=OFF"
+)
```

Comment:
I'd kinda prefer switching to newer flang already (copy [this](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/flang19.yaml) under `.ci_support/migrations` and rerender), because then you can just do the same logic as in the unix script (e.g. test if `FC` is not empty).

Reply:
I can do that. I was planning to do that in #100 as soon as this is merged, but I can merge it here if that's better.

Reply:
this now includes the flang19 migration and uses the simpler `"%FC%" NEQ ""` condition